cpp/oneapi/dal/algo/pca/backend/gpu/train_kernel_cov_impl_dpc.cpp

```diff
@@ -104,7 +104,7 @@ auto compute_covariance(sycl::queue& q,
 
     auto copy_event = copy(q, cov, xtx, { deps });
 
-    auto cov_event = pr::covariance(q, row_count, sums, cov, { copy_event });
+    auto cov_event = pr::covariance(q, row_count, sums, cov, false, { copy_event });
```

Comment:
Please use non hardcoded value here

Reply:
There should be false value to keep the PCA algorithm unchanged